### ISSUE TITLE
More A11y Violations Detected

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#26390]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

A11y violations detected by cypress dashboard

### Version

23.0.0

### Expected behavior

Violations are fixed

### Actual behavior

Violations need to be fixed 

### How to Reproduce?

Several violations

**Comment by jonkoops** [comment]
@agagancarczyk I see you reported this issue for v22, but we are currently on 23, is this issue still relevant.

**Comment by agagancarczyk** [comment]
> @agagancarczyk I see you reported this issue for v22, but we are currently on 23, is this issue still relevant.

Sorry, I made a mistake in specifying the version. I have now changed it to 23.
